devnotes/mvpd_generation.md

```diff
@@ -0,0 +1,20 @@
+# MVPD data modification
+
+## Testing environement
+
+Original `PNOR` image was aquired by building `op-build`.
+
+`MVPD` partition was read using `pflash` tool:
+```
+pflash -P MVPD -r <output file>
+```
+## Test results
+
+* After flashing original pnor image, `MVPD` partition
```

Comment:
The results are very restrained. Please provide some logs/outputs and comment them rather than providing a comment only.

Reply:
Added screenshots and links to `MVPD` dumps